src/perform_step/low_order.jl

```diff
@@ -230,3 +230,50 @@ end
 
   end
 end
+
+#methods from https://doi.org/10.1016/j.cam.2009.11.010
+@muladd function perform_step!(integrator,cache::WangLi3SMil_AConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  #stage 1
+  k  = integrator.f(uprev,p,t)
+  u  = uprev + dt*k
+
+  #stage 2
+  k  = integrator.g(u,p,t)
+  tmp = u + k*integrator.sqdt
+  k₁ = integrator.g(tmp,p,t)
+  k₁ = (k₁ - k)/(integrator.sqdt)
+  u  = u - (dt/2)*k*k₁
+
+  #stage 3
+  k  = integrator.g(u,p,t)
+  tmp = u + k*integrator.sqdt
+  k₁ = integrator.g(tmp,p,t)
```

Comment:
t+dt

Reply:
@ChrisRackauckas This is from a previous paper by the same author describing 2-stage Milstein methods
![image](https://user-images.githubusercontent.com/26359940/58683149-fde9d480-8390-11e9-939f-201a326b47c6.png)
There they assume g = g(t,u) and they use g(t,Yn)

Reply:
but you're not writing what they are doing in the paper since you're doing a derivative approximation. We could require that the user pass gg' (http://docs.juliadiffeq.org/latest/solvers/sde_solve.html#Derivative-Based-Methods-1) but I'm telling you what needs to be done for the 1st order approximation of that